extensions/airpods-noise-control/package.json

```diff
@@ -2,28 +2,110 @@
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "airpods-noise-control",
   "title": "AirPods Noise Control",
-  "description": "Toggle between Noise Cancellation and Transparency on your AirPods.",
+  "description": "Switch between Noise Cancellation, Transparency or Adaptive on your AirPods. Toggle Conversation Awareness on AirPods that support it.",
   "icon": "extension_icon.png",
   "author": "chrahe",
+  "contributors": [
+    "hllvc"
+  ],
   "categories": [
     "Media",
     "System"
   ],
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
+      "name": "switch-noise-control",
```

Comment:
Would be better to keep the same index if a command is the same, this way people don't lose a hotkey or ranking